modules/dnsmasq_dhcp/autodetection.go

```diff
@@ -6,28 +6,42 @@ import (
 	"io/ioutil"
 	"net"
 	"os"
-	"path"
+	"path/filepath"
 	"regexp"
 	"strings"
 )
 
+// configDir represents conf-dir directive
+//
+// #conf-dir=/etc/dnsmasq.d
+//
+// # Include all the files in a directory except those ending in .bak
+// #conf-dir=/etc/dnsmasq.d,.bak
+//
+// # Include all files in a directory which end in .conf
+// #conf-dir=/etc/dnsmasq.d/,*.conf
+type configDir struct {
+	path          string
+	includeSuffix []string
+	excludeSuffix []string
```

Comment:
> 	includeSuffix []string
> 	excludeSuffix []string

Why slice?

> conf-dir=/etc/dnsmasq.d/,*.conf

is `conf-dir=/etc/dnsmasq.d/,*.conf, *.cconf` a valid syntax?

Reply:
Yes, it's valid syntax. I have not found it in dnsmasq's documentation, but found it in dnsmasq's source code, here is the link to dnsmasq's github mirror:

https://github.com/imp/dnsmasq/blob/4e7694d7107d2299f4aaededf8917fceb5dfb924/src/option.c#L1560

I've also checked it practically and it works exactly like that.